packages/server/src/templates/projects/auth/passwordReset.tsx

```diff
@@ -10,34 +10,35 @@ export interface ForgotPasswordProps {
 export default function (props: ForgotPasswordProps) {
 
     return (
-        <Container component="main" maxWidth="xs">
-            <Box
-                sx={{
-                    marginTop: 10,
-                    display: 'flex',
-                    flexDirection: 'column',
-                    alignItems: 'center',
-                }}
-            >
-                <Typography component="h1" variant="h5">
+      <EmailTemplate title="Password Reset">
+        <Box
+            sx={{
+                marginTop: 10,
+                display: 'flex',
+                flexDirection: 'column',
+                alignItems: 'center',
+            }}
+        >
+            <Typography component="h1" variant="h5">
+                Reset Password
+            </Typography>
+            <br></br>
```

Comment:
No need to add breaks, just wrap the div below and add padding or margin-top, same goes for below